polkadot/node/core/pvf/src/prepare/worker_intf.rs

```diff
@@ -70,7 +70,7 @@ pub async fn spawn(
 /// If the idle worker token is not returned, it means the worker must be terminated.
 pub enum Outcome {
 	/// The worker has finished the work assigned to it.
-	Concluded { worker: IdleWorker, result: PrepareResult },
+	Concluded { worker: IdleWorker, result: PrepareResult, path: Option<PathBuf> },
```

Comment:
It's better to put the path in `PrepareResult`. The path should only exist in the `Ok` case, then it also doesn't have to be an `Option`.

Reply:
Oh, I remember now. Coz `PrepareResult` is propagated from somewhere the path (renamed to) is not available. Now that I think of it, maybe I could just use a placeholder.